app.js

```diff
@@ -141,6 +141,7 @@ app.use(expressErrorLogger);
 
 // eslint-disable-next-line
 app.use((err, req, res, next) => {
+  logger.error(err.stack);
```

Comment:
a couple lines above this you should see
```
app.use(expressErrorLogger);
```

This should log any errors and then send them along for further processing (such as by THIS error handler). This is configured in `./winston.js` and perhaps we need to update this configuration to provided better logging of errors?